swift/Tests/HDWalletTests.swift

```diff
@@ -420,4 +420,9 @@ class HDWalletTests: XCTestCase {
 
         XCTAssertEqual(address, "band1pe8xm2r46rmctsukuqu7gl900vzprfsp4sguc3")
     }
+
+    func testBip39Suggest() {
+        XCTAssertEqual(HDWallet.bip39Suggest(prefix: "air"), "air airport")
```

Comment:
should we return as an array of words? `["air", "airdrop"]` ?

Reply:
this can be done by adding a swift extension

Reply:
```swift
extension Mnemonic {
    func suggest() -> [String]
}
```

Reply:
My arguments for using space-separated string:
- simplifies a lot crossing C++/C/JNI interfaces
- it is easy to split to array on other side
- mnemonic phrase is also sent as space-separated string to wallet core

Reply:
on top of this, we can even return all wrong words + index for a whole mnemonic

Reply:
@catenocrypt let's use string in c/cpp and create swift/kotlin native type extensions

Reply:
type extension: I've got no experience with this, do we have already sg like this?

Reply:
> on top of this, we can even return all wrong words + index for a whole mnemonic

Let's keep some logic to the client as well ;)  These calls are in-process function calls, no delay like for a BE API

Reply:
I will add that for you

Reply:
@hewig, it is also possible that all words are valid, but the phrase not (due to checksum).

Reply:
I have added Mnemonic.isWordValid(), so words can be checked one-by-one

Reply:
that's possible, if last word is treated as wrong checksum